nss-day-cohort-45/hello-world-column-counts
# Ar navbar design

## changes:
1) designed and styled navbar
2) added import for nav.css to main.css

## test:
1) make sure hover on navbar links work and styling looks right
2) make sure links appear to link to correct pages by testing in browser
